json/scripts/make-json.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+# Script to create the Json checklist file for WSTG
+
+cd document/4-Web_Application_Security_Testing
+
+# Start the json string
+echo "{" > checklist.json
+# Open categories
+echo "    \"categories\": {"  >> checklist.json
+# Iterate all category folders to get the categories
+for d in */ ; do
+    # Split the folder name to get category and category number
+    # Select folders start form 01
+    while IFS='-' read -ra FOLD; do
+        if [ ${FOLD[0]} -gt 0 ]
+        then
+            category=${FOLD[1]}
+            # Start category sub section
+            # Add coma from the second entry of the list
+            if [ ${FOLD[0]} -gt 1 ];then
+                echo "        ,\"${category%?}\": {" | tr '_' ' '  >> checklist.json
+            else
+                echo "        \"${category%?}\": {" | tr '_' ' '  >> checklist.json
+            fi
+            # Get category ID from the first file
+            cid=`cat $d/01-* | grep "|WSTG-.*" | cut -d "|" -f 2 | sed 's/-01//'`
+            # Add category ID and start tests
+            echo "            \"id\":\"${cid}\","  >> checklist.json
+            echo "            \"tests\":["  >> checklist.json
+            count=0
+            for file in $d*.md; do
+                # Remove README
+                if [[ $file != *"README.md" ]];then
+                    # Add coma from the second entry of the list
+                    if [ $count -gt 0 ];then
+                        echo "                ,{"  >> checklist.json
+                    else
+                        echo "                {"  >> checklist.json
+                    fi
+                    # Get test ID, test name and reference link from the file
+                    tid=`cat $file | grep "|WSTG-.*" | cut -d "|" -f 2`
+                    tname=$(head -n 1 $file)
```

Comment:
From a small research, I believe this is a better format.
```suggestion
                    read -r tname < $file
```